src/main/java/com/paskar/email/application/repositiory/EmailStorage.java

```diff
@@ -0,0 +1,49 @@
+package com.paskar.email.application.repositiory;
+
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.paskar.email.application.console.Email;
+import org.springframework.stereotype.Component;
+
+import java.io.*;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@Component
+public class EmailStorage implements EmailRepository {
+    private final static String baseFile = "emailList.json";
+
+    @Override
+    public void save(List<Email> email) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+        PrintWriter out = new PrintWriter(new BufferedWriter(
+                new FileWriter(baseFile, true))); // append mode file writer
+        mapper.writeValue(out, email);
+    }
+
+    @Override
+    public Email createNewEmail() throws IOException {
+        return null;
+    }
+
+    @Override
+    public List<Email> findAll() throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
You can reuse ObjectMapper here and in save method. Just declare it as spring bean(using custom config) and inject it here instead of creating new instance